SuperblocksHQ/superblocks-lab
# Add new template - CryptoPizza

### Description of the Change
New template called CryptoPizza. Simple Crypto-collectible game built on top of ERC-721 standard. User can create his own Pizza based on name (string) which generates random DNA (uint) that shows up as random Pizza. 


### Designs

![screen1](https://user-images.githubusercontent.com/17637244/49725770-cd8c6d80-fc6c-11e8-86e9-4cbafa75ec33.PNG)
![screen2](https://user-images.githubusercontent.com/17637244/49725771-ce250400-fc6c-11e8-80dc-236a21eeff16.PNG)


### Verification Process
1. Create new project CryptoPizza
2. Compile and Deploy
3. Open `appview` and create random Pizza
4. Pizza should be shown in `Inventory`
5. User should be able to transfer Pizza to other wallet and burn the Pizza



### Github Issues
